ubuntu22.04/precompiled/Dockerfile

```diff
@@ -65,7 +65,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends nvidia-driver-$
 # avoid cleaning the cache after this to retain these packages during runtime.
 RUN apt-get update && apt-get install --download-only --no-install-recommends -y linux-objects-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION} \
     linux-signatures-nvidia-${KERNEL_VERSION} \
-    linux-modules-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION}
+    linux-modules-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION} \
+    # add support for nvidia open source driver packages during runtime
+    linux-objects-nvidia-${DRIVER_BRANCH}-server-open-${KERNEL_VERSION} \
```

Comment:
```suggestion
```

Reply:
As discussed on slack, we need the `linux-objects-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION}` package.

Reply:
updated the code. 
removed linux-objects-nvidia-${DRIVER_BRANCH}-**server-open**-${KERNEL_VERSION} \

Reply:
We need to add to add the `linux-objects-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION}` package back.

Reply:
Ah nevermind, I see that is still being installed.